From 5fd5e42bf28a7910321c6b6d76257e7386839fbc Mon Sep 17 00:00:00 2001
From: Michael Weghorn <m.weghorn@posteo.de>
Date: Wed, 16 Feb 2022 13:38:47 +0100
Subject: configure.ac: Update kf5 include/lib check to work with KF5 >= 5.91

The 'kcoreaddons_version.h' header was moved to the 'KCoreAddons'
subdirectory in kcoreaddons commit

    commit d971ba9bb27a3e8f18a116692fdf98c1729ac244
    Author: Ahmad Samir <a.samirh78@gmail.com>
    Date:   Sat Jan 15 14:14:13 2022 +0200

        Install kcoreaddons_version.h in /usr/include/KF5/KCoreAddons/

        Instead of /usr/include/KF5/kcoreaddons_version.h.

        For more details see:
        https://invent.kde.org/frameworks/kservice/-/merge_requests/79

        GIT_SILENT

Adapt the KF5 check to check for the 'KFileWidget' header and
the 'libKF5KIOFileWidgets.so' library instead.
This is more exact anyway, since we require KIOFileWidgets,
and the previous check didn't ensure that.
(So in case only KF5CoreAddons but not KIOFileWidgets was installed,
configure would previously pass, but the build fail.)

[1] https://invent.kde.org/frameworks/kcoreaddons/-/commit/d971ba9bb27a3e8f18a116692fdf98c1729ac244

Change-Id: I7ecf7f29dce85e1a6e6b0d4f2519fa37ab04ca84
Reviewed-on: https://gerrit.libreoffice.org/c/core/+/129996
Tested-by: Jenkins
Reviewed-by: Michael Weghorn <m.weghorn@posteo.de>
---
 configure.ac | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index 6acb3107cea3..4e84f9a33980 100644
--- a/configure.ac
+++ b/configure.ac
@@ -13017,8 +13017,8 @@ then
         kf5_libdirs="$kf5_libdirs /usr/lib64 /usr/lib64/kf5 /usr/lib64/kf5/devel"
     fi
 
-    kf5_test_include="KF5/kcoreaddons_version.h"
-    kf5_test_library="libKF5CoreAddons.so"
+    kf5_test_include="KF5/KIOFileWidgets/KFileWidget"
+    kf5_test_library="libKF5KIOFileWidgets.so"
     kf5_libdirs="$qt5_libdir $kf5_libdirs"
 
     dnl kf5 KDE4 support compatibility installed
-- 
cgit v1.2.1

